crates/nargo_cli/tests/test_data_ssa_refactor/option/Nargo.toml

```diff
@@ -0,0 +1,5 @@
+[package]
```

Comment:
This test should be moved to `test_data` now right? I believe you will also have to add a `name` field.